### PR TITLE
issue: #3190

### PR DESCRIPTION
**Pull request by Anshika-G**
Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.

Please delete anything that isn't relevant to your patch.

* [x] This PR relates to an existing open issue and there are no existing
      PRs open for the same issue.
* [ ] Add `Fixes #ISSUENUM` at the top of your PR.
* [ ] Add a description of the the changes introduced in this PR.
* [ ] The change has been successfully run locally.
* [ ] Add tests to cover the changes added in this PR.
* [ ] Add before and after screenshots (Only for changes that impact the UI).

Once you have met the above requirements please replace this section with
a `Fixes #ISSUENUM` linking to the issue fixed by this PR along with an
explanation of the changes. Thanks for your contribution!

[
ss_before
](url)Before committing changes
ss_afterAfter committing changes

### CODE REVIEW

**Comment by tofumatt** [comment]
Sorry, but you’ll need to attach screenshots of the site itself. You can install the site using node (either npm or yarn) and then run it locally to test your changes.

There are docs in the README and the docs/ folder of the repo outlining how to do this.

Matthew Riley MacPherson (sent from mobile)

**Comment by Anshika-G** [comment]
sure 👍  @tofumatt

File: src/ui/components/Card/Card.scss [hunk 1]
```diff
@@ -41,7 +41,7 @@ $footer-padding: 10px;
 
   border-bottom-left-radius: $border-radius-default;
   border-bottom-right-radius: $border-radius-default;
-  font-variant: all-small-caps;
+
```
Comment (tofumatt) [inline src/ui/components/Card/Card.scss]: This empty line can just be removed, by the way. That way you won't get the linting test failure.

Comment (Anshika-G) [reply src/ui/components/Card/Card.scss]: Yeah, I know that but Im unable to run docker for testing the changes.

Comment (tofumatt) [reply src/ui/components/Card/Card.scss]: You don’t need docker to run this app locally, just node 6: https://github.com/mozilla/addons-frontend

Once you install node you can develop against the dev site using`yarn amo:dev`.

**Comment by muffinresearch** [comment]
The  default PR description has a list of requirements for a new PR - please could you take a look at that and follow the steps?  I'll add it back to the description.  You can delete it once you've ticked all the boxes.

**Pull request closed without merging**